src/client/components/Header.jsx

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
 
+import '../styles/header.css'
+
 const Header = () => {
     return (
-        <>
-        </>
+        <div className='header-container'>
+            <h1 className='header-header'>Bookr</h1>
+            <div className='links-container'>
+                <p>Home</p>
```

Comment:
I'm expecting `Link` here, eventually ;)

Reply:
It will be!